client/a8c-for-agencies/sections/sites/sites-dashboard/index.tsx

```diff
@@ -57,10 +57,15 @@ export default function SitesDashboard() {
 		selectedSiteUrl,
 		selectedSiteFeature,
 		setSelectedSiteFeature,
+		isFavoriteFilter,
 		selectedCategory: category,
 		setSelectedCategory: setCategory,
 	} = useContext( SitesDashboardContext );
 
+	// TODO: this is just an example
+	// eslint-disable-next-line no-console
+	console.log( ' is_favorite param = ' + isFavoriteFilter );
```

Comment:
We need to remove this before merging it.

Reply:
We should remove this debug code

Reply:
Removed in https://github.com/Automattic/wp-calypso/pull/88919/commits/d3e22e8673bee3eb56573d262e1605154250ab93